shelley/design-spec/delegation_design_spec.tex

```diff
@@ -2718,32 +2747,108 @@ \subsection{Reward Splitting}
+invites freeriders, pools that rely on other pools to maintain the system, and
+collect rewards without doing any work.
+
+For Cardano, we will stay close to the scheme of~\citep{bkks2018}, but we will
+also take into account the \emph{performance} of a pool. Unfortunately, tracking
+the performance of a pool is not trivial in protocols without a public leader
+schedule, like Ouroboros Praos.
+
+In order to get a handle on the performance of a pool, let us consider the
+fraction \(\beta\) of all blocks within a given epoch that the pool created.
+
+There are multiple factors that influence \(\beta\): the stake of the pool, how
+well it performs, and some randomness due to the leader election and the overall
+performance of the other pools and the network. We can write\footnote{We assume
+  a linear relation between stake and the number of blocks that a pool is
+  eligible to create. Strictly speaking, this is not the case for Ouroboros
+  Praos. However, the error when linearising the leader election function is
+  small, particularly for the range of parameters we are considering for
+  Cardano.}
+\[\beta = \sigma p r_e \,,\]
+where
+\begin{enumerate}
+\item[\(\sigma\)] is the relative stake \(\sigma\) of the pool (the fraction of
+  the absolute stake that the pool controls),
+\item[\(p\)] is the performance \(p\) of the pool, i.e. the fraction
+  \[
+  p = \frac{n}{\max(N,1)}
+  \]
+  of the number \(n\) blocks it successfully added to the chain and the number
+  \(N\) of slots it was elected as a leader, and
+\item[\(r_e\)] is a factor that captures the relation between the relative stake
+  \(\sigma\) of the pool, the number \(N\) of slots it is elected as a leader,
+  and the total number \(\overline{N}\) of blocks that were added to the chain during
+  the epoch. To be precise, we have
+  \[r_e = \frac{N}{\sigma \overline{N}}\,.\]
+  The factor \(r_e\) captures all the random influences on \(\beta\): the
+  randomness in the leader election that influences \(n\), and the randomness
+  both from the leader election and the bunch of random influences on
+  \(\overline{N}\) (leader election, forks, performance of the network and other
+  pools).
+\end{enumerate}
+
+If we insert the definitions of \(p\) and \(r_e\) into \(\beta\), we find that
+\[
+\beta = \sigma \frac{n}{\max(N,1)} \frac{N}{\sigma \overline{N}}
+= \frac{n}{\overline{N}}\,,
+\]
+which is indeed the fraction of blocks produced by the pool\footnote{When
+  cancelling \(N\) and \(\max(N,1)\), note that for \(N=0\), also \(n=0\), and
+  the equation trivially holds.}.
+
+In Ouroboros Praos, we can observe \(\sigma\), \(n\), and \(\overline{N}\), but
+we do not have a direct handle on any of \(N\), \(r_e\), or \(p\). From the
+observables, we can extract
+\[ \beta = \frac{n}{\overline{N}}\,. \]
+While the true performance \(p\) is not accessible, we can define and measure
+the \emph{apparent performance}
```

Comment:
To me it is not clear (from the formulae above) why this performance is "apparent". I'd expect that "apparent" in this context means "approximation" (but maybe I got this wrong). If that's the case, then I wonder where are we loosing precision, since we replaced equals by equals in the derivation of `\pbar`.

Reply:
It's the product of the actual performance of the pool and all the random influences on the number of blocks that the pool produces -- their luck or bad luck in the leader election, network effects, ...

The reason I called it apparent performance is that if you are oblivious to the randomness in leader election and assume a perfect network, the pool would appear to have this performance.

Does that make things clearer :)

Reply:
Oh, I see. Thanks.